Makefile

```diff
@@ -77,7 +85,9 @@ build-cpu-py-37-base:
 	docker build -f Dockerfile-base-cpu \
 		--build-arg BASE_IMAGE="$(UBUNTU_IMAGE_TAG)" \
 		--build-arg PYTHON_VERSION="$(PYTHON_VERSION_37)" \
-		--build-arg "$(MPI_BUILD_ARG)" \
+		--build-arg UBUNTU_VERSION="ubuntu20.04" \
+        --build-arg "$(MPI_BUILD_ARG)" \
+		--build-arg "$(OFI_BUILD_ARG)" \
```

Comment:
I think this resolves to an empty string, for example, here: https://app.circleci.com/pipelines/github/determined-ai/environments/363/workflows/27d11f93-746e-44a2-8720-6f604d882e77/jobs/2798

```
#!/bin/bash -eo pipefail
make build-tf1-cpu NGC_PUBLISH= REGISTRY_REPO=environments-dev WITH_MPI=1
docker build -f Dockerfile-base-cpu \
	--build-arg BASE_IMAGE="ubuntu:20.04" \
	--build-arg PYTHON_VERSION="3.7.11" \
	--build-arg UBUNTU_VERSION="ubuntu20.04" \
        --build-arg "USE_GLOO=1" \
	--build-arg "" \
	-t determinedai/environments-dev:py-3.7-base-cpu-6ade8bf \
	-t determinedai/environments-dev:py-3.7-base-cpu-0.19.1 \
	.
invalid argument "" for "--build-arg" flag: invalid environment variable: 
See 'docker build --help'.
make: *** [Makefile:85: build-cpu-py-37-base] Error 125

Exited with code exit status 2
CircleCI received exit code 2
```